### ISSUE TITLE
Statcast Game Date Format Problem

### ISSUE THREAD

**Issue by jldbc** [jldbc/pybaseball#104]
Did the statcast csv change its date format? Someone tagged me in this on Twitter https://twitter.com/ckurcon/status/1301913190465507328

I get: `ValueError: time data "2020-09-03T00:00:00.000Z" doesn't match format specified`

**Comment by TheCleric** [comment]
Doesn't appear to be the case for me. I have a statcast file from 9/1 in my saves that shows date like "2020-09-01", and I just ran a couple files with past and current/future dates and see the same format.

**Comment by TheCleric** [comment]
This is my copy of the repo, but is synced with master (which is admittedly slightly out of date with the released version, but nothing around Statcast or dates to my knowledge):

```
Python 3.8.5 (tags/v3.8.5:580fbb0, Jul 20 2020, 15:57:54) [MSC v.1924 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import pybaseball
>>> pybaseball.statcast('2020-08-25', '2020-09-03').head()
This is a large query, it may take a moment to complete
Completed sub-query from 2020-08-25 to 2020-08-30
Completed sub-query from 2020-08-31 to 2020-09-03
  pitch_type  game_date  ...  if_fielding_alignment  of_fielding_alignment
0         SL 2020-09-03  ...               Standard               Standard
1         SL 2020-09-03  ...               Standard               Standard
2         SI 2020-09-03  ...               Standard               Standard
3         FS 2020-09-03  ...          Infield shift              Strategic
4         SI 2020-09-03  ...          Infield shift              Strategic

[5 rows x 89 columns]
```

**Comment by jldbc** [comment]
Maybe they fixed it? I got the error an hour ago but don't get it anymore

**Comment by TheCleric** [comment]
Strange. In the mentioned ticket above we can try to support both date formats in the new try_parse parsing function and cut Statcast parsing over to that function (which would also fix #59 as well).

Maybe I'll tackle that next.

**Comment by jldbc** [comment]
Looks like this was a one-off thing, I'm going to close this. 

Some [additional context](https://twitter.com/darenw/status/1301987012191367168) from darenw (doesn't explain why this was happening, but looks like it was unintended behavior that shouldn't be showing up again)
